### ISSUE TITLE
Specifying the algorithm?

### ISSUE THREAD

**Issue by XhmikosR** [neftaly/handlebars-helper-sri#3]
@neftaly: So, I was trying to specify the algorithm to use in srihash.org and I couldn't get it to work; I end up with the default sha256.

**Comment by neftaly** [comment]
On this repo or on srihash? The repo for srihash is https://github.com/mozilla/srihash.org

**Comment by XhmikosR** [comment]
Yes, I know :) I meant, I was trying to change to sha384 for srihash.org, which is using your helper.
